File: Modul 1/Bahasa C/PRAK108-2310817210005-GALIH_AJI_SABDARAYA.c
#include <stdio.h>

int main(){
    int putaran = 5, jarak_tempuh = 14;
    float pi = 3.14;
    float keliling_lingkaran = (float)jarak_tempuh / putaran;
    float jari_jari = (float)keliling_lingkaran / (2 * pi);
    printf("Diketahui :\n");
    printf("Pak Denglek mengelilingi taman = %d Putaran\n", putaran);
    printf("Jarak tempuh Pak Dengklek = %d Kilometer\n\n", jarak_tempuh);
    printf("Jawaban :\n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer", jari_jari);
    return 0;
}

File: Modul 1/Soal 8/PRAK108-2310817210005-GALIH_AJI_SABDARAYA.py
import math
putaran = 5
jarak_tempuh = 14
keliling_lingkaran = jarak_tempuh / putaran
jari_jari = round(keliling_lingkaran / (2 * math.pi ), 2)
print("Diketahui :")
print(f"Pak Denglek mengelilingi taman = {putaran} Putaran")
print(f"Jarak tempuh Pak Dengklek = {jarak_tempuh} Kilometer\n")
print("Jawaban :")
print(f"Jari-jari taman yang dikelilingi Pak Dengklek adalah {jari_jari} Kilometer")